embuilder.py

```diff
@@ -58,7 +58,6 @@
 ]
 
 USER_TASKS = [
-    'binaryen',
```

Comment:
is this an unrelated change?

Reply:
Yeah those are part of a separate change now.

Reply:
https://github.com/emscripten-core/emscripten/pull/11585